__tests__/flow.ts

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+const { spawnSync } = require('child_process');
+
+test('it detects errors', () => {
+  const flowBin = path.resolve(__dirname, '../node_modules', '.bin', 'flow');
```

Comment:
The `.cmd` file muse be executed for Windows users.

```suggestion
  const flowBin = path.resolve(__dirname, '../node_modules', '.bin', process.platform === 'win32' ? 'flow.cmd' : 'flow',);
```